desktop/app/splash.js

```diff
@@ -1,9 +1,9 @@
 // Checks to see if we've launched before (we touch started.txt). If not, write the file and open a browser
-import {socketRoot} from '../../react-native/react/constants/platform.native.desktop'
+import {socketRoot, dataRoot} from '../../react-native/react/constants/platform.native.desktop'
 import {helpURL} from '../../react-native/react/constants/urls'
 import path from 'path'
 
-const filePath = path.join(socketRoot, 'started.txt')
+const filePath = path.join(process.platform === 'win32' ? dataRoot : splashRoot, 'started.txt')
```

Comment:
we should just expose a different variable from the constants that does the check there.